test/org/pcic/DockerUtilsTest.groovy

```diff
@@ -0,0 +1,101 @@
+package org.pcic
+
+import util.MockSteps
+
+import org.junit.Before
+import org.junit.Test
+
+
+public class DockerUtilsTest {
+    DockerUtils dockerUtils
+
+    @Before
+    public void init() {
+        MockSteps mockSteps = new MockSteps()
+        this.dockerUtils = new DockerUtils(mockSteps)
+    }
+
+    @Test
+    public void determineTags_masterAndGitTags() {
+        // prepare
+        String branchName = 'master'
+        def gitTags = ['1.0.0', 'some-tag']
+        ArrayList expected_tags = ['1.0.0', 'some-tag', 'latest']
+        int expected_size = 3
+
+        // execute
+        ArrayList result = dockerUtils.determineTags(branchName, gitTags)
+
+        // assert
+        assert result == expected_tags
+        assert result.size() == expected_size
+    }
```

Comment:
Is the commenting pattern `// prepare` ... `// execute` ... `// assert` typical or idiomatic in Jenkins/groovy? The comments seem unnecessary; the whitespace already separates the three blocks adequately, and also `assert` calls out the test expectations proper. I see this throughout.